docs/GUIdemo/my_yml.yml

```diff
@@ -0,0 +1,41 @@
+# Example YAML configuration for TARDIS
```

Comment:
Can you make this a symbolic link that points to `docs/tardis_example.yml`, and then it is best if the link is called `tardis_example.yml` for consistency.